module/x/gravity/keeper/ethereum_event_vote.go

```diff
@@ -244,30 +256,9 @@ func (k Keeper) getLastEventNonceByValidator(ctx sdk.Context, validator sdk.ValA
 		// them to replay the entire history of all events ever we can't start
 		// at zero
 		//
-		// We could start at the LastObservedEventNonce but if we do that this
-		// validator will be slashed, because they are responsible for making a claim
-		// on any attestation that has not yet passed the slashing window.
-		//
-		// Therefore we need to return to them the lowest attestation that is still within
-		// the slashing window. Since we delete attestations after the slashing window that's
-		// just the lowest observed event in the store. If no claims have been submitted in for
-		// params.SignedClaimsWindow we may have no attestations in our nonce. At which point
-		// the last observed which is a persistent and never cleaned counter will suffice.
+		// We return the LastObservedEventNonce
 		lowestObserved := k.GetLastObservedEventNonce(ctx)
-		attmap := k.GetEthereumEventVoteRecordMapping(ctx)
```

Comment:
Removed this section. 

May be some old code? 

The validator arent actually getting slashed for not submitting ethereum events and we are not pruning the event neither